Microsoft.DotNet.Interactive.Formatting/Formatter.cs

```diff
@@ -311,6 +356,11 @@ public static void ResetToDefault()
             SingleLinePlainTextFormatter.WriteEndSequence(writer);
         }
 
+        public static IEnumerable<string> MimeTypesFor(Type type)
```

Comment:
This is a bit leaky conceptually because formatters can register lazily.

Reply:
mmm